File: backend-portifolio/Models/Post.cs
using System.ComponentModel.DataAnnotations;
namespace backend_portifolio.Models
{
    public class Post
    {
        [Key]   
        public long Id { get; set; }
        public Guid Guid {get; set;}
        public List<Section> Sections {get; set;}

        public Post()
        {
            this.Sections = new List<Section>();
            this.Guid = Guid.NewGuid();
        }
    }
}

File: portfolio-v3/src/domain/models/Post.ts
import { Category } from "./Category";
import { DevelopmentTool } from "./DevelopmentTool";
import { Like } from "./Like";
import { Link } from "./Link";
import { Section } from "./Section";
import { Comment } from "./Comment";

export class Post{
    public Id: number;
    public Guid: string;
    public Sections: Section[];
    public Categories: Array<Category>;
    public DevelopmentTools : Array<DevelopmentTool>;
    public DateCreate: Date;
    public Links: Array<Link>;
    public Likes: Array<Like>;
    public Comments: Comment[];
    constructor(){
        this.Id = 0;
        this.Guid = "";
        this.Sections = new Array<Section>();
        this.Categories = new Array<Category>();
        this.DateCreate = new Date();
        this.DevelopmentTools = new Array<DevelopmentTool>();
        this.Links = new Array<Link>();
        this.Likes = new Array<Like>();
        this.Comments = new Array<Comment>()
    }
}